src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -863,6 +865,36 @@ public ItemStack get(PlayerEditBookEvent e) {
 				return book;
 			}
 		}, 0);
-		
+		//ItemDespawnEvent
+		EventValues.registerEventValue(ItemDespawnEvent.class, Item.class, new Getter<Item, ItemDespawnEvent>() {
+			@Override
+			@Nullable
+			public Item get(ItemDespawnEvent e) {
+				return e.getEntity();
+			}
+		}, 0);
+		EventValues.registerEventValue(ItemDespawnEvent.class, ItemStack.class, new Getter<ItemStack, ItemDespawnEvent>() {
+			@Override
+			@Nullable
+			public ItemStack get(ItemDespawnEvent e) {
+				return e.getEntity().getItemStack();
+			}
+		}, 0);
+		//ItemMergeEvent
+		//TODO there is also e.getTarget() two entities involved in this event, currently can be worked around currently with `on item merge of (insert target itemtype)`
```

Comment:
I'd say an expression that acts as event value for this event would be enough

Reply:
Assigning both Item and ItemStack, but no ItemType... That is an interesting choice. We will need to make sure it is tested well.